packages/react/src/components/DataTable/TableBatchActions.js

```diff
@@ -58,13 +59,13 @@ const TableBatchActions = ({
       className={batchActionsClasses}
       {...rest}>
       <div className={batchSummaryClasses}>
-        <p className={`${prefix}--batch-summary__para`}>
+        <Text as="p" className={`${prefix}--batch-summary__para`}>
```

Comment:
I initially thought it would be best to place it on the element with the classname but having it on the innermost node is probably preferable. I'll change it